src/js/port_handler.js

```diff
@@ -120,8 +118,15 @@ PortHandler.initialize = function () {
         }
     }, MDNS_INTERVAL);
 
-    // start listening, check after TIMEOUT_CHECK ms
-    this.check();
+    this.reinitialize();    // just to prevent code redundancy
+};
+
+PortHandler.reinitialize = function () {
+    this.initialPorts = false;
+    clearTimeout(this.check_loop);
```

Comment:
```suggestion
    clearTimeout(this.usbCheckLoop);
```

Reply:
Sorry, I missed it when renamed the variable.